test/api-spec.js

```diff
@@ -168,4 +168,8 @@ describe('api', function () {
 
     assert.deepStrictEqual(topLevelFunctions, defaultExportFunctions);
   });
+  it('should stat a symlinked file', async () => {
+    const stats = asar.statFile('test/input/stat-symlink.asar', 'real.txt', true);
+    return assert.strictEqual(stats.size, 19);
```

Comment:
Can we assert that this is not a link instead of checking the size? I think it makes the intent clearer:

```suggestion
    return assert.strictEqual(stats.link, undefined);
```

Reply:
Good point. Done.